README.md

```diff
@@ -24,10 +24,10 @@ requests in React.
 
 ### Features
 
-✓ Uses the native [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) API  
-✓ Smart [deduping of requests](./docs/guides/request-deduplication.md)  
-✓ Customizable [response caching](./docs/guides/response-caching.md)  
-✓ Provides hooks to integrate with external stores (like [Redux](https://github.com/reactjs/redux))  
```

Comment:
These extra whitespaces will need to stick around for the formatting to remain the same 🙂